### PR TITLE
DS 690-search-unit-tests

### PR DESCRIPTION
**Pull request by zuzannanap**

### CODE REVIEW

File: packages/design-system/src/svgs/SearchSvg.tsx [hunk 1]
```diff
@@ -8,6 +8,9 @@ const SearchSvg: FC<ClassName> = ({ className }) => {
       fill="none"
       xmlns="http://www.w3.org/2000/svg"
       className={className}
+      aria-hidden
+      role="img"
+      data-testid="searchSvg"
```
Comment (monikaracka) [inline packages/design-system/src/svgs/SearchSvg.tsx:13]: I don't think we should add test id to icons.

File: packages/design-system/src/components/Search/__tests__/Search.screen-object.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import Search from "..";
+import { SearchProps } from "../Search.interface";
+
+export const mockErrorMessage = "Error";
+export class SearchObject {
+  get input() {
+    return screen.getByPlaceholderText("Placeholder");
+  }
+  get icon() {
+    return screen.getByTestId("searchSvg");
```
Comment (monikaracka) [inline packages/design-system/src/components/Search/__tests__/Search.screen-object.tsx:12]:
```suggestion
    return screen.getByRole("img", { hidden: true });
```

**Review by monikaracka — APPROVED** [review]

File: packages/design-system/src/svgs/SearchSvg.tsx (hunk 1 above)
Comment (zuzannanap) [reply packages/design-system/src/svgs/SearchSvg.tsx]: hmm weird i don't have it on local branch 🙃

Comment (zuzannanap) [reply packages/design-system/src/svgs/SearchSvg.tsx]: oh ok, i haven't see you commented it before i removed it, nvm :D

**Pull request merged**